internal/apps/dop/providers/issue/dao/issue_property.go

```diff
@@ -75,6 +77,41 @@ func (client *DBClient) GetIssueProperties(req pb.GetIssuePropertyRequest) ([]Is
 		return nil, err
 	}
 
+	// propertiesProject 项目级 properties
+	var propertiesProject []IssueProperty
+	overlayIndex := make([]int, 0)
+	if req.ScopeType != "project" {
+		goto come
+	}
+
+	if req.ScopeType != "" {
+		db = db.Where("scope_type = ?", req.ScopeType)
+	}
+	if req.ScopeId != "" {
+		db = db.Where("scope_id = ?", req.ScopeId)
+	}
+	if err := db.Order("index").Find(&propertiesProject).Error; err != nil {
+		return nil, err
+	}
+	// 优先级：项目 > 企业级，当有重复的字段时，项目覆盖企业的字段；
+
```

Comment:
remove this empty line.
extract the logic to a function and add unit-test.

Reply:
ok
very reasonable
thanks